src/main/java/com/virtudoc/web/service/AuthenticationService.java

```diff
@@ -78,6 +78,7 @@ public void RegisterNewAccount(UserAccount user) throws Exception {
      * Updates a UserAccount with a new password. Performs password hashing before saving.
      * @param user Existing UserAccount object with new password stored in password field.
      */
+    @throws Exception exception
```

Comment:
Sorry, my bad for not explaining this. The `@throws` should be inside the block comment. The full line would be `* @throws Exception exception` between linkes 79 and 80.

Reply:
Done